web/source/osk/oskLayer.ts

```diff
@@ -85,13 +90,16 @@ namespace com.keyman.osk {
     public refreshLayout(vkbd: VisualKeyboard, paddedHeight: number, trueHeight: number) {
```

Comment:
So, fun detail.  Even after the prior PR introduced these objects... turns out this code path was only ever used for touch keyboards.  This PR's changes to these methods allow `refreshLayout` to handle _both_ types, rather than just touch keyboards.